packages/dx-react-grid-bootstrap4/src/templates/table-tree-cell.test.jsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { TableTreeCell } from './table-tree-cell';
+
+describe('TableTreeCell', () => {
+  it('should render children if passed', () => {
+    const tree = shallow((
+      <TableTreeCell>
+        <span className="test" />
+      </TableTreeCell>
+    ));
+
+    expect(tree.find('.test').exists())
+      .toBeTruthy();
+  });
+
+  it('should pass style to the root element', () => {
+    const tree = shallow((
+      <TableTreeCell
+        style={{ color: 'gray' }}
```

Comment:
Its test is excess because you already have 'should pass rest props to the root element'.